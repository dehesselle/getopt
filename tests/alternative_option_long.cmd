#!/bin/sh
"$1" --alternative -o a -l one,two:,three::,four:: -- -a -one -two=MANDATORY -three=OPTIONAL
