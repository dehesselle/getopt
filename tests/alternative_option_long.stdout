 -a --one --two 'MANDATORY' --three 'OPTIONAL' --
